proto/meta.proto

```diff
@@ -204,21 +215,20 @@ message DeleteWorkerNodeResponse {
   common.Status status = 1;
 }
 
-message ListAllNodesRequest {
-  common.WorkerType worker_type = 1;
-  // Whether to include nodes still starting
-  bool include_starting_nodes = 2;
+// This request should NOT delete the node, but just cordon it aka. mark it as unschedulable
+message CordonWorkerNodeRequest {
+  common.HostAddress host = 1;
 }
 
-message ListAllNodesResponse {
+message CordonWorkerNodeResponse {
   common.Status status = 1;
-  repeated common.WorkerNode nodes = 2;
 }
 
 service ClusterService {
   rpc AddWorkerNode(AddWorkerNodeRequest) returns (AddWorkerNodeResponse);
   rpc ActivateWorkerNode(ActivateWorkerNodeRequest) returns (ActivateWorkerNodeResponse);
   rpc DeleteWorkerNode(DeleteWorkerNodeRequest) returns (DeleteWorkerNodeResponse);
+  rpc CordonWorkerNode(CordonWorkerNodeRequest) returns (CordonWorkerNodeResponse);
```

Comment:
Can we make it more generic, like using a more universal function, such as `UpdateWorkerNode`, so that we can perform both cordon and uncordon operations?